automatic/git/git.nuspec

```diff
@@ -6,29 +6,27 @@
     <version>2.11.0</version>
     <authors>The Git Development Community</authors>
     <owners>Chocolatey</owners>
-    <summary>Git (for Windows) – We bring the awesome Git SCM to Windows</summary>
+    <summary>Git or Windows offers a native set of tools that bring the full feature set of the Git SCM to Windows</summary>
     <description>
 Git for Windows focuses on offering a lightweight, native set of tools that bring the full feature set of the Git SCM to Windows while providing appropriate user interfaces for experienced Git users and novices alike.
 
-## Features
```

Comment:
Why remove the feature section as this is required by the guidelines ([$1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description)